gateway-shell/src/test/java/org/apache/knox/gateway/shell/table/KnoxShellTableTest.java

```diff
@@ -385,11 +385,15 @@ public void testMean() throws Exception {
       Long long20 = (long) 200000000;
       Long long30 = (long) 300000000;
 
+      String string10 = "-10";
+      String string20 = "-20";
+      String string30 = "-30.00";
```

Comment:
Need to test positives? Also would do more and less than two digits. Probably makes sense to randomize this. Random and then multiply to make a double. Same for the other values

Reply:
Ah right, my bad, meant to change that test back to positives and negatives before pushing it up. Thanks for catching that. See updated patch.